resource-types/GithubStatus.dhall

```diff
@@ -0,0 +1,52 @@
+{ meta = { name = "github-status", repository = "resource/github-status" }
+, Version = { Type = {}, default = {=} }
+, Params =
+  { Get =
+    { Type = { path : Text, context : Optional Text }
+    , default.context = None Text
+    }
+  , Put =
+      let StatusObject =
+            { Type =
+                { state : Text
+                , context : Optional Text
+                , description : Optional Text
+                , target_url : Optional Text
+                }
+            , default =
+              { context = None Text
+              , description = None Text
+              , target_url = None Text
+              }
+            }
+
+      let Statuses =
+            { Type = { path : Text, statuses : List StatusObject.Type }
+            , default = {=}
+            }
+
+      let Status =
+            { Type = { path : Text } ⩓ StatusObject.Type
+            , default =
+              { context = None Text
+              , description = None Text
+              , target_url = None Text
+              }
+            }
+
+      let Put =
+              { Type = < Status : Status.Type | Statuses : Statuses.Type > }
+            ∧ { StatusObject, Status, Statuses }
```

Comment:
See comment above, I would prefer to export `StatusObject` under `Statuses` (which is the only place it is relevant) rather than other `Put`. but this is not a big deal